README.md

```diff
@@ -1,5 +1,7 @@
 ## Serto UI
 
+[![SertoID](https://circleci.com/gh/SertoID/serto-ui.svg?style=svg&circle-token=1c9f5796a3867357f99158ec5200124191b126eb)](https://circleci.com/gh/SertoID/serto-ui/tree/master)
```

Comment:
This looks like a duplicate of the existing CircleCI logo below but a different style and with the token, presumably we should use this and delete the other.